diffblue.yml

```diff
@@ -5,20 +5,61 @@ phases:
  timeout: 300
  cbmcArguments:
    classpath: '/tools/cbmc/models-simple-overlay.jar:/tools/cbmc/models.jar:.'
-   max-nondet-array-length: 100
-   java-max-vla-length: false
+   java-assume-inputs-non-null: true
+   java-max-vla-length: 48
+   max-nondet-array-length: 10
+   max-nondet-string-length: 10
+   string-printable: true
```

Comment:
I think this is set to `true` in phase 1 on a lot of benchmarks because in phase 1 we're aiming for pretty tests? Not objecting to changing it, just thinking we may want to check with whoever made the decision to set it to `true` here.
I'm guessing the same applies to `java-assume-inputs-non-null`?

Reply:
Yes, both these options seem to be for pretty tests. They should not really decrease coverage though as far as I understand (`java-assume-inputs-non-null` is only used in the first phase, so the missing tests can still use null later on).